apps/judicial-system/backend/src/app/messages/pdfCourtRecord.ts

```diff
@@ -56,6 +56,13 @@ export const courtRecord = {
     defaultMessage: 'Mættir eru:',
     description: 'Notaður sem fyrirsögn á lista yfir viðstadda.',
   }),
+  nonePresentInSession: defineMessage({
```

Comment:
This Contentful string is not necessary as the text will come from the client autofilled field `courtAttendees`.